tests/ignite/contrib/handlers/test_tqdm_logger.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 import sys
 import time
+from packaging.version import Version
```

Comment:
About packaging, it is not a built-in package right ? Seems like minimal pytorch-ignite installation does not require it neither. Currently, we use `from distutils.version import LooseVersion` in the code. What do you think ?